Jenkinsfile

```diff
@@ -12,6 +12,9 @@
 //   5. You should get an editor where you can modify the pipeline and run it
 //      again immediately <3
 
+def dockerRepoName = 'zooniverse/front-end-monorepo-${APP_ENV}'
```

Comment:
I'm not sure that APP_ENV is available at this point, it looks to be declared in the `environment` block below.

Reply:
Good catch.

Reply:
The Jenkins build logs have the correct image name. I'm going to move this line anyway, because this is confusing.
```
+ docker build -t zooniverse/front-end-monorepo-staging:316fd973152192300e5f859399e0d142b19681bb --build-arg APP_ENV --build-arg COMMIT_ID --build-arg CONTENTFUL_ACCESS_TOKEN --build-arg CONTENTFUL_SPACE_ID --build-arg CONTENT_ASSET_PREFIX --build-arg SENTRY_CONTENT_DSN --build-arg PROJECT_ASSET_PREFIX --build-arg SENTRY_PROJECT_DSN .
Sending build context to Docker daemon  48.72MB
```
https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/job/PR-1887/28/console